app/lib/search/search_service.dart

```diff
@@ -550,15 +550,15 @@ class TagsPredicate {
   /// user-facing search query.
   Map<String, String> asSearchLinkParams() {
     final params = <String, String>{
-      'runtime': _tagPartsWithPrefix('runtime', value: true).join(' '),
-      'platform': _tagPartsWithPrefix('platform', value: true).join(' '),
+      'runtime': tagPartsWithPrefix('runtime', value: true).join(' '),
+      'platform': tagPartsWithPrefix('platform', value: true).join(' '),
```

Comment:
Something missing here?

Reply:
Removed the leading `_` in the name, as I'm using outside of the class.

Reply:
nvm